tests/acceptance/features/coreApiWebdavPreviews/previews.feature

```diff
@@ -367,3 +367,39 @@ Feature: previews of files downloaded through the webdav API
       | spaces           | fill      | fill.png       |
       | spaces           | resize    | resize.png     |
       | spaces           | thumbnail | thumbnail.png  |
+
+  @issue-10589 @env-config
+  Scenario Outline: try to download an image preview when the maximum thumbnail input value in the environment is set to a small value
+    Given the following configs have been set:
+      | config                               | value |
+      | THUMBNAILS_MAX_INPUT_IMAGE_FILE_SIZE | 1KB   |
+      | THUMBNAILS_MAX_INPUT_WIDTH           | 200   |
+      | THUMBNAILS_MAX_INPUT_HEIGHT          | 200   |
+    And using <dav-path-version> DAV path
+    And user "Alice" has uploaded file "filesForUpload/testavatar.jpg" to "/testimage.jpg"
+    When user "Alice" downloads the preview of "/testimage.jpg" with width "32" and height "32" and processor thumbnail using the WebDAV API
+    Then the HTTP status code should be "403"
+    And the value of the item "/d:error/s:message" in the response about user "Brian" should be "thumbnails: image is too large"
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
```

Comment:
@nirajacharya2 this scenario give 200

Reply:
your CI build is failing. Please adjust it according to your comment 
![Screenshot from 2024-12-24 16-39-18](https://github.com/user-attachments/assets/c692bae5-c8d8-481b-ac2b-0ae7473761af)